lib/pbench/server/utils.py

```diff
@@ -72,7 +72,7 @@ def quarantine(dest, logger, *files):
         sys.exit(101)
 
     for afile in files:
-        if not os.path.exists(afile) and not os.path.islink(afile):
+        if not os.path.exists(afile):
```

Comment:
Since we are updating this condition, can we also update it to use `Pathlib` module something like `if not 
 Path(afile).exists()`